app/javascript/shared/components/focusTrap.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import { h, Fragment } from 'preact';
+import { useEffect, useRef } from 'preact/hooks';
+import { createFocusTrap } from 'focus-trap';
+import { defaultChildrenPropTypes } from '../../common-prop-types';
+
+const FocusTrap = ({ selector, children }) => {
```

Comment:
Is it worth having a `bool` active prop?
Meaning in the future we could add the ability to lock focus if we want to,

Reply:
I think boolean `active` can be added to **FocusTrap** component and made optional default value being `true`. But right now it is not used here but it could be added later to the component if needed without affecting existing code.